R/pkg/R/generics.R

```diff
@@ -521,7 +521,7 @@ setGeneric("sampleBy", function(x, col, fractions, seed) { standardGeneric("samp
 
 #' @rdname saveAsTable
 #' @export
-setGeneric("saveAsTable", function(df, tableName, source, mode, ...) {
```

Comment:
`setGeneric("saveAsTable", function(df, tableName, source = NULL, mode = "error", ...) { `       is better? since it indicates more precise signature.